extension/jaegerremotesampling/internal/http.go

```diff
@@ -58,9 +58,6 @@ func NewHTTP(telemetry component.TelemetrySettings, settings confighttp.HTTPServ
 	}
 
 	srv.mux = http.NewServeMux()
-	// the legacy endpoint
-	srv.mux.Handle("/", http.HandlerFunc(srv.samplingStrategyHandler))
-
 	// the new endpoint -- not strictly necessary, as the previous one would match it
```

Comment:
may want to fix the comment: "the previous one" doesn't exist anymore

Reply:
Ups, good catch :X